editor/selectors.js

```diff
@@ -826,6 +826,16 @@ export function isMultiSelecting( state ) {
 	return state.blockSelection.isMultiSelecting;
 }
 
+/**
+ * Whether is selection disable or not.
```

Comment:
As having trouble remembering what it means to have "selected enabled", given the various types of selections which can occur in Gutenberg, I found this comment insufficient in explaining what I can expect as the meaning of the value returned by this selector.